openedx/features/caliper_tracking/apps.py

```diff
@@ -1,41 +1,20 @@
 from django.apps import AppConfig
 from django.conf import settings
 
-from .settings import (
-    CALIPER_TRACKING_APP_NAME,
-    CALIPER_TRACKING_BACKENDS,
-    CALIPER_TRACKING_PROCESSOR,
-    LOGIN_EVENT_EMITTER
-)
+from .settings import LOGIN_EVENT_EMITTER
 
 
 class CaliperTrackingConfig(AppConfig):
-    name = CALIPER_TRACKING_APP_NAME
+    name = 'openedx.features.caliper_tracking'
```

Comment:
Why is the full path necessary?

Reply:
right now this path is necessary, but once we transform it into a separate app and move it into site-packages then it won't be necessary